OpenRA.Mods.Common/Traits/Render/WithCargoPipsDecoration.cs

```diff
@@ -69,7 +69,8 @@ string GetPipSequence(int i)
 				{
 					var sequence = Info.FullSequence;
 					if (pi.CustomPipType != null && !Info.CustomPipSequences.TryGetValue(pi.CustomPipType, out sequence))
-						Log.Write("debug", "Actor type {0} defines a custom pip type {1} that is not defined for actor type {2}".F(c.Info.Name, pi.CustomPipType, self.Info.Name));
+						Log.Write("debug", "Tick {3}: Actor type {0} defines a custom pip type {1} that is not defined for actor type {2}"
```

Comment:
This would ideally be caught by a lint test. Nobody actually reads debug.log afaik so we could delete this too and leave it up to modders to realize that the game is falling back to the default pip sequence to identify that something is wrong.